stimela/__init__.py

```diff
@@ -1,9 +1,11 @@
 import os
 from pathlib import Path
-
-__author__ = """Sphesihle Makhathini, Oleg Smirnov and RATT"""
-__email__ = "sphemakh@gmail.com"
-__version__ = "2.0rc2"
+#-----------
+# knicked from https://github.com/python-poetry/poetry/issues/273#issuecomment-1103812336
```

Comment:
I think you mean "nicked". The other spelling means something else: https://www.urbandictionary.com/define.php?term=knicked